bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -24,22 +24,41 @@
 import static com.google.cloud.bigtable.config.CallOptionsConfig.SHORT_TIMEOUT_MS_DEFAULT;
```

Comment:
Can we split this class into classic & veneer variants? The only thing they need to share are the string constants for the keys, which can be extracted to a common class.
It'll make it easier to compare the 2 implementations